### PR TITLE
Config layout cleanup

### PR DESCRIPTION
**Pull request by joydeep-b**

### CODE REVIEW

File: config/ut_jackal.lua [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  odometry_topic = "/jackal_velocity_controller/odom";
+  initialpose_topic = "/initialpose";
+};
+
+enml = {
+  -- On the cobblestone next to the GDC door closest to the lab,
+  -- ********************************************************
+  -- DEFAULT STARTING LOCATION
+  -- On the cobblestone next to the GDC door closest to the lab,
+  -- right next to grass, on the side with the lamp post and facing Speedway.
```
Comment (kvsikand) [inline config/ut_jackal.lua:14]: I agree that this config file probably isn't right place to put it, but this drops a lot of the commented out information for where the jackal started in various bag files.

**Review by kvsikand — APPROVED** [review]

**Comment by maxsvetlik** [comment]
I've finally got a change to test this on `jackal`. It works, and I'm happy over all. However @joydeep-b, based on the quantity of config files in `config/` it looks like we're moving away from the `robot.example` local implementation, yes?

**Review by maxsvetlik — APPROVED** [review]

**Comment by joydeep-b** [comment]
Yup @maxsvetlik good point - we should have only as many configs as needed to have examples off different types of robots. How about keeping just the following examples, to document in the Readme:
1. Jackal, as example of skid-steer robot with VLP-16
2. CoBot, as example of Hokuyo UST-10lx with an omnidirectional drive
3. UT AUTOmata, as example of Hokuyo UST-10lx with ackermann steering drive, on limited computation (Jetson TX2)
4. Spot, as example of VLP-16 on a quadruped

**Comment by maxsvetlik** [comment]
I think that's a good idea. This repository is not the place to maintain config files for various platforms.
However I do see the value in having the config files written for platforms easily accessible. They can then be used for multiagent simulations with different platforms, without having to rewrite them from scratch.

BWI has [this](https://github.com/utexas-bwi/ahg_common) repo for shared AHG data across labs. BWI currently uses it for occupancy maps, but it would also house AHG's robot configs. I haven't made up my mind if it makes sense for the configs to be hosted within the robot's respective codebase, or somewhere central like `ahg_common`. If you have an opinions, let me know.

**Comment by joydeep-b** [comment]
I think if you are storing configs for a homogeneous set of robots, which are all used for one project in one lab, a single repo is best - that is what we did when maintaining four CoBots.

**Pull request merged**